execute/plugin.go

```diff
@@ -562,18 +589,72 @@ func (p *Plugin) Reports(seqNr uint64, outcome ocr3types.Outcome) ([]ocr3types.R
 func (p *Plugin) ShouldAcceptAttestedReport(
 	ctx context.Context, u uint64, r ocr3types.ReportWithInfo[[]byte],
 ) (bool, error) {
-	panic("implement me")
```

Comment:
Basic implementations for ShouldAcceptAttestedReport and ShouldTransmitAcceptedReport